detox/local-cli/detox-test.js

```diff
@@ -63,12 +63,12 @@ function runJest() {
   cp.execSync(command, {
     stdio: 'inherit',
     env: Object.assign({}, process.env, {
-      configuration: program.configuration,
-      loglevel: program.loglevel,
-      cleanup: program.cleanup,
-      reuse: program.reuse,
-      debugSynchronization: program.debugSynchronization,
-      artifactsLocation: program.artifactsLocation
+      configuration: program.configuration || '',
```

Comment:
Can you stick to defaults that `commander` use?

Reply:
Because the flags are `''` the defaults will be used by the commander - similar to mocha.

Reply:
I'm just saying that instead of using `|| ''` everywhere in the `execSync` command you can specify defaults to 
`program.option`s in the beginning of the file. It supports three parameters in `.option` chain:

```js
program.option('-flag', 'description', 'defaults')
```

so your `|| ''` can be re-written in a more elegant way. For instance,

```js
program.option('-l, --log-level', 'Log level ...', '')
```

where third `''` is a default value.

Does it make sense?